roles/kubernetes/secrets/tasks/check-certs.yml

```diff
@@ -101,7 +106,8 @@
       {% if gen_node_certs[inventory_hostname] or
         (not kubecert_node.results[0].stat.exists|default(False)) or
           (not kubecert_node.results[10].stat.exists|default(False)) or
```

Comment:
`kubecert_node` now contains N+2 items, because of two new keys, isn't we checking wrong keys now?